docs/_source/components-overview/pop_syn/population_params.rst

```diff
@@ -459,16 +808,44 @@ The :class:`~posydon.binary_evol.binarystar` class contains the binary systems a
 the parameters here determine what output of that class will be outputted into the final population file.
 
 .. list-table::
-   :widths: 50 50
-   :header-rows: 1
-
-   * - Parameter
-     - Description/Value
-   * - `extra_columns`
-     - {'step_names':'string', 'step_times':'float64'} ('step_times' with from posydon.binary_evol.simulationproperties import TimingHooks)
-   * - `only_select_columns`
-     - ['state', 'event', 'time', 'orbital_period', 'eccentricity', 'lg_mtransfer_rate'] (all options: 'state', 'event', 'time', 'separation', 'orbital_period', 'eccentricity', 'V_sys', 'rl_relative_overflow_1', 'rl_relative_overflow_2', 'lg_mtransfer_rate', 'mass_transfer_case', 'trap_radius', 'acc_radius', 't_sync_rad_1', 't_sync_conv_1', 't_sync_rad_2', 't_sync_conv_2', 'nearest_neighbour_distance')
-                      
+  :widths: 10 80 10
+  :class: population-params-table
+  :header-rows: 1
+
+  * - Parameter
+    - Description
+    - Default Value
+
+  * - ``extra_columns``
+    - | Additional columns to include in the output.
+      | Example: ``{'step_names':'string', 'step_times':'float64'}``
+      | Note: ``'step_times'`` requires ``TimingHooks`` from ``posydon.binary_evol.simulationproperties``.
+    - See default ``population_params.ini``.
```

Comment:
Here you refer to the `population_params_default.ini` (missing the default). It should be consistent with the other tables: either reference or value. In the case of a reference, please link the file; in case of a value may consider to read it from the file.